examples/ad7124.py

```diff
@@ -37,22 +37,21 @@
 from scipy import signal
 
 # Set up AD7124
-ad7124 = adi.ad7124(uri="ip:192.168.1.171")
+ad7124 = adi.ad7124(uri="ip:10.48.65.138")
+ad_channel = 0
 
 sc = ad7124.scale_available
-ad7124.channel[0].scale = sc[2]
+
+ad7124.channel[ad_channel].scale = sc[-1]  # get highest range
 ad7124.rx_output_type = "SI"
 
-# sets sample rate for all channels
-ad7124.sample_rate = 19200
-# currently only one enabled channel buffer works at a time
-ad7124.rx_enabled_channels = [0]
+ad7124.sample_rate = 19200  # sets sample rate for all channels
+ad7124.rx_enabled_channels = [
+    ad_channel
+]  # currently only one enabled channel buffer works at a time
```

Comment:
If this is a part limitation can you provide an overloaded rx_enabled_channel set property then?

Reply:
It used to be, now it should work with multiple channels. The performance is better/worse based on the sample rates of the channels.